.github/pull_request_template.md

```diff
@@ -11,8 +11,9 @@
 * [ ] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
 * [ ] This pull request is all my own work -- I have not plagiarized.
 * [ ] I know that pull requests will not be merged if they fail the automated tests.
-* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
+* [ ] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
 * [ ] All new Javascript files are placed inside an existing directory.
-* [ ] All filenames follow camel casing style.
+* [ ] All filenames follow kebab-case style.There should be no spaces in filenames.<br/>
+     **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
```

Comment:
is this ok, @cclauss

Reply:
https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/ ;-)